src/delete.ts

```diff
@@ -0,0 +1,182 @@
+			price: 189,
+			salePrice: 119,
+			rating: 4,
+			numberOfReviews: 735,
+		},
+		{
+			name: "6L Digital Roaster Air Fryer with Paper Liners",
+			imageUrl:
+				"https://media.takealot.com/covers_images/055772c8fe06464e8258d4af33a67132/s-fb.file",
+			productLink:
+				"https://www.takealot.com/6l-digital-roaster-air-fryer-with-paper-liners/PLID93227833",
+			price: 1,
+			salePrice: 589,
+			rating: 4.4,
+			numberOfReviews: 858,
+		},
+		{
+			name: "TRESemme Care and Protect Heat Protection Spray Hair Treatment 300ml",
+			imageUrl:
+				"https://media.takealot.com/covers_images/86230a6ae18d4a66b6030be2bf36d417/s-fb.file",
+			productLink:
+				"https://www.takealot.com/tresemme-care-and-protect-heat-protection-spray-hair-treatment-300ml/PLID41295269",
+			price: 119,
+			salePrice: 97,
+			brand: "Tresemme",
+			brandLink: "https://www.takealot.com/all?filter=Brand:Tresemme",
+			rating: 4.7,
+			numberOfReviews: 1111,
+		},
+		{
+			name: "TWS Wireless Earphones i12",
+			imageUrl:
+				"https://media.takealot.com/covers_images/9cf3bc2a1fca400893a9e514072bcfbe/s-fb.file",
+			productLink:
+				"https://www.takealot.com/tws-wireless-earphones-i12/PLID90233146",
+			price: 118,
+			rating: 3.8,
+			numberOfReviews: 1882,
+		},
+		{
+			name: "10 in 1 Baby Care Kit",
+			imageUrl:
+				"https://media.takealot.com/covers_tsins/55532033/55532033-1-fb.jpg",
+			productLink:
+				"https://www.takealot.com/10-in-1-baby-care-kit/PLID525845759",
+			price: 91,
+			rating: 4.7,
+			numberOfReviews: 1681,
+		},
+		{
+			name: "Mondi Rotatrim A4 Office Copy Paper Ream White 500 Sheets",
+			imageUrl:
+				"https://media.takealot.com/covers_tsins/40876850/6003977000190-1-fb.jpg",
+			productLink:
+				"https://www.takealot.com/mondi-rotatrim-a4-office-copy-paper-ream-white-500-sheets/PLID38148850",
+			price: 190,
+			salePrice: 120,
+			brand: "Mondi",
+			brandLink: "https://www.takealot.com/all?filter=Brand:Mondi",
+			rating: 4.7,
+			numberOfReviews: 379,
+		},
+		{
+			name: "Super Fast 45W Charging Wall Charger & Type C to Type C Cable - Black",
+			imageUrl:
+				"https://media.takealot.com/covers_images/7195d7c502c34c69ae90487eb99630da/s-fb.file",
+			productLink:
+				"https://www.takealot.com/super-fast-45w-charging-wall-charger-type-c-to-type-c-cable-black/PLID90807249",
+			price: 200,
+			salePrice: 134,
+			brand: "H-Tech",
+			brandLink: "https://www.takealot.com/all?filter=Brand:H-Tech",
+			rating: 3.4,
+			numberOfReviews: 838,
+		},
+		{
+			name: "World Map Anti-Slip Extended Mouse Pad",
+			imageUrl:
+				"https://media.takealot.com/covers_images/62547a7d91ac4c02a86a6e31dd10adbc/s-fb.file",
+			productLink:
+				"https://www.takealot.com/world-map-anti-slip-extended-mouse-pad/PLID707227390",
+			price: 99,
+			rating: 4.5,
+			numberOfReviews: 1536,
+		},
+	];
+
+	const browser = await chromium.launch();
+	const context = await browser.newContext();
+
+	for (let product of products) {
```

Comment:
You can use ...`const product`... here since product is not re-assigned inside the body

Reply:
Thank you!